ragoon/models/base.py

```diff
@@ -49,6 +55,12 @@ class Embed:
     model: Optional[str] | Optional[list[str]] = (
         "sentence-transformers/all-MiniLM-L6-v2"
     )
+    embedder: Optional[str] = EmbedderType.DEFAULT_CHROMA_EMBEDDER.value
+    docs_embedding_count: Optional[int] = 10
+
+    def __post_init__(self):
+        if self.embedder not in [e.value for e in EmbedderType]:
+            raise ValueError("Wrong embedder specified")
```

Comment:
@gkaretka let's mention the Embedder name in the error message too, if possible, and maybe also mention the supported options?